docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
+- https://docs.openshift.com/container-platform/4.3/rest_api/index.html#serviceaccount-v1-core
 
-* Location of serviceaccounts in archive: config/serviceaccounts
-* See: docs/insights-archive-sample/config/serviceaccounts
-* Id in config: clusterconfig/service_accounts
-* Since versions:
-  - 4.5.34+
-  - 4.6.20+
-  - 4.7+
+### Sample data
+- [docs/insights-archive-sample/config/serviceaccounts.json](./insights-archive-sample/config/serviceaccounts.json)
+
+### Location in archive
+- `config/serviceaccounts.json`
+
+### Config ID
+`clusterconfig/service_accounts`
+
+### Released version
+- 4.7.0
+
+### Backported versions
+- 4.5.34+
+- 4.6.20+
+
+### Changes
+None
 
 
 ## SilencedAlerts
 
-gathers the alerts that have been silenced.
+Collects the alerts that have been silenced.
+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/config/silenced_alerts.json](./insights-archive-sample/config/silenced_alerts.json)
+
+### Location in archive
+- `config/silenced_alerts.json`
+
+### Config ID
+`config/silenced_alerts`
 
-* Location in archive: config/silenced_alerts
-* See: docs/insights-archive-sample/config/silenced_alerts
-* Id in config: clusterconfig/silenced_alerts
-* Since version:
-  - 4.10+
+### Released version
+- 4.10.0
+
+### Backported versions
+None
+
+### Changes
+None
 
 
 ## StorageCluster
 
 Collects `storageclusters.ocs.openshift.io` resources
-from Openshift Data Foundation Stack.
 
-### API Reference:
-https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/storagecluster_types.go
+### API Reference
+- https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/storagecluster_types.go
 
 ### Sample data
-docs/insights-archive-sample/config/storage/openshift-storage/storageclusters/ocs-storagecluster.json
+- [docs/insights-archive-sample/config/storage/openshift-storage/storageclusters/ocs-storagecluster.json](./insights-archive-sample/config/storage/openshift-storage/storageclusters/ocs-storagecluster.json)
 
-### Location in archive:
-| Version | Path														|
-| ------- | --------------------------------------------------------	|
-| <= 4.12 | config/storage/{namespace}/{name}.json 					|
-| 4.13+   | config/storage/{namespace}/storageclusters/{name}.json 	|
+| Version   | Path														|
+| --------- | --------------------------------------------------------	|
+| < 4.12.0  | config/storage/{namespace}/{name}.json 					|
+| >= 4.12.0 | config/storage/{namespace}/storageclusters/{name}.json 	|
 
-### Config ID:
-clusterconfig/storage_cluster
+### Config ID
+`clusterconfig/storage_cluster`
 
-### Since versions:
-* 4.11
+### Released version
+- 4.11.0
+
+### Backported versions
+None
+
+### Changes
+- Renamed from `OpenshiftStorage` to `StorageCluster` in version `4.12.0+`
+- Config ID changed from `clusterconfig/openshift_storage` to `clusterconfig/storage_cluster` in version `4.12.0+`
 
```

Comment:
I would like to add a counterpart to the disclaimer for CephCluster (suggested in another comment):

```suggestion
- In OCP 4.11 and OCP 4.12, the location of gathered data collides with data gathered by the [GatherCephCluster](#cephcluster) gatherer. It is practically impossible to tell the two resources apart. Use with caution.
```